src/api/app/controllers/webui/user_controller.rb

```diff
@@ -73,14 +74,17 @@ def save
     @displayed_user = User.find_by_login(params[:user][:login])
 
     unless User.current.is_admin?
-      if User.current != @displayed_user
+      if User.current != @displayed_user || !@configuration.accounts_editable?
         flash[:error] = "Can't edit #{@displayed_user.login}"
         redirect_back(fallback_location: root_path)
         return
       end
     end
 
-    @displayed_user.assign_attributes(params[:user].slice(:realname, :email).permit!)
+    unless @configuration.ldap_enabled?
```

Comment:
This is necessary since admins and user can `POST` to that action. Since an admin can still POST to this action and change at least the roles of a user, the admin should not be able to change the user's real name nor the email address.